### ISSUE TITLE
Keyboard commands not registering

### ISSUE THREAD

**Issue by serosenstein** [Shell-Company/go-scout#7]
Love that you added keyboard support. However, I'm not able to get it to work yet.

I've tried on same subnet as well as different subnets.
I've tried issuing command while docked and undocked.
The X11 Video window does appear and is streaming.
When I press space bar I CAN get a screenshot to save. 
 
No errors are experienced using WASD, **but no movement occurs**

When I press keys like H or 0 or 9 it causes the panic seen below:
[steve@t470 go-scout]$ ./scout -h 10.1.99.11:11311 -l 10.1.99.46 -c keyboard -v
Starting go-scout controller for Moorebot Scout
ROS endpoint: 10.1.99.11:11311
2023/02/16 12:03:45 creating camera access node using 10.1.99.11:11311
panic: service returned a failure state

goroutine 23 [running]:
main.turnOnLight(0x1)
        /home/steve/opt/go-scout/lights.go:55 +0x2af
main.robotControlKeyboard()
        /home/steve/opt/go-scout/main.go:291 +0x1b3
created by main.main
        /home/steve/opt/go-scout/main.go:124 +0x533

**Comment by serosenstein** [comment]
To add: switching to "Branch" branch I was able to get "H" key to work and scoutbot returns home.  "Command received: Scout returning home in 5 seconds".  That and screenshot work in this branch, and 0/9 don't cause it to panic but doesn't seem to do anything.

**Comment by serosenstein** [comment]
So the keyboard commands were getting registered after I added some debug statements in.  Then I thought the X and Y params weren't getting set and I printed those out too.  That was also ok.  Then I printed the final geometry sent to the Twist API thingy. It was changing when I pressed the key.  So then I did a wireshark capture of the traffic.  I noticed that the requests send the sender's IP and it was using random ports.  I run firewalld and I block traffic between certain subnets/VLANs. on my router  So I put both devices back on the same subnet /VLAN and I turned off my host based firewall (firewalld) and finally I got this to work!

Suggestion: When using verbose flag print out more info

Thanks so much for this project! This is awesome.
